src/topology/algebra/multilinear.lean

```diff
@@ -56,12 +55,12 @@ namespace continuous_multilinear_map
 section semiring
 
 variables [semiring R]
-[∀i, add_comm_monoid (M i)] [∀i, add_comm_monoid (M₁ i)] [add_comm_monoid M₂] [add_comm_monoid M₃]
-[add_comm_monoid M₄]
-[∀i, semimodule R (M i)] [∀i, semimodule R (M₁ i)] [semimodule R M₂] [semimodule R M₃]
-[semimodule R M₄]
-[∀i, topological_space (M i)] [∀i, topological_space (M₁ i)] [topological_space M₂]
-[topological_space M₃] [topological_space M₄]
+[Πi, add_comm_monoid (M i)] [Πi, add_comm_monoid (M₁ i)] [Πi, add_comm_monoid (M₁' i)]
```

Comment:
Is there a rule of thumb stylistically between `Π` and `∀`? (also, should there be a space after the binder?)

Reply:
The rule is to add a space after binders. I think the tendency is to use `∀` in preference to `Π`, but I can't find it codified somewhere.
```suggestion
[Π i, add_comm_monoid (M i)] [Π i, add_comm_monoid (M₁ i)] [Π i, add_comm_monoid (M₁' i)]
```

Reply:
We try to use `∀` for predicates, `Π` for data.